### PR TITLE
Regression: Fix incoming voip call ringtone is not ringing

### PR DESCRIPTION
**Pull request by murtaza98**

### CODE REVIEW

File: client/providers/CallProvider/CallProvider.tsx [hunk 1]
```diff
@@ -16,6 +19,19 @@ import { useUser } from '../../contexts/UserContext';
 import { roomCoordinator } from '../../lib/rooms/roomCoordinator';
 import { isUseVoipClientResultError, isUseVoipClientResultLoading, useVoipClient } from './hooks/useVoipClient';
 
+const newCallSound = (user: IUser): void => {
```
Comment (amolghode1981) [inline client/providers/CallProvider/CallProvider.tsx:22]: Can we rename this to startRingback()

File: client/providers/CallProvider/CallProvider.tsx [hunk 2]
```diff
@@ -16,6 +19,19 @@ import { useUser } from '../../contexts/UserContext';
...
+	const audioVolume = getUserPreference(user, 'notificationsSoundVolume');
+	CustomSounds.play('telephone', {
+		volume: Number((audioVolume / 100).toPrecision(2)),
+		loop: true,
+	});
+};
+
+const stopCallSound = (): void => {
```
Comment (amolghode1981) [inline client/providers/CallProvider/CallProvider.tsx:30]: Can we rename this to stopRingback()

**Review by amolghode1981 — CHANGES_REQUESTED** [review]

File: client/providers/CallProvider/CallProvider.tsx (hunk 2 above)
Comment (murtaza98) [inline client/providers/CallProvider/CallProvider.tsx:30]:
```suggestion
const stopRingback = (): void => {
```

File: client/providers/CallProvider/CallProvider.tsx [hunk 3]
```diff
@@ -181,6 +197,10 @@ export const CallProvider: FC = ({ children }) => {
 
 		const { registrationInfo, voipClient } = result;
 
+		voipClient.on('incomingcall', () => user && newCallSound(user));
+		voipClient.on('callestablished', () => stopCallSound());
```
Comment (murtaza98) [inline client/providers/CallProvider/CallProvider.tsx:201]:
```suggestion
		voipClient.on('callestablished', () => stopRingback());
```

File: client/providers/CallProvider/CallProvider.tsx [hunk 4]
```diff
@@ -181,6 +197,10 @@ export const CallProvider: FC = ({ children }) => {
...
+		voipClient.on('callterminated', () => stopCallSound());
```
Comment (murtaza98) [inline client/providers/CallProvider/CallProvider.tsx:202]:
```suggestion
		voipClient.on('callterminated', () => stopRingback());
```

File: client/providers/CallProvider/CallProvider.tsx [hunk 5]
```diff
@@ -181,6 +197,10 @@ export const CallProvider: FC = ({ children }) => {
 
 		const { registrationInfo, voipClient } = result;
 
+		voipClient.on('incomingcall', () => user && newCallSound(user));
```
Comment (murtaza98) [inline client/providers/CallProvider/CallProvider.tsx:200]:
```suggestion
		voipClient.on('incomingcall', () => user && startRingback(user));
```

File: client/providers/CallProvider/CallProvider.tsx (hunk 1 above)
Comment (murtaza98) [inline client/providers/CallProvider/CallProvider.tsx:22]:
```suggestion
const startRingback = (user: IUser): void => {
```

Comment (murtaza98) [reply client/providers/CallProvider/CallProvider.tsx]: For sure, its done

**Review by amolghode1981 — APPROVED** [review]

**Pull request merged**